kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateList.js

```diff
@@ -67,59 +75,103 @@ define([
         );
     }
 
+    function createTableRow(tableBody, id) {
+        const newRow = document.createElement('tr');
+        newRow.setAttribute('data-element-job-id', id);
+        newRow.classList.add(`${cssBaseClass}__row`);
+        return tableBody.appendChild(newRow);
+    }
+
     // Convert the table to a datatable object to get functionality
-    function renderTable($container, numberOfChildJobs) {
-        $container.find('table').dataTable({
+    function renderTable($container, rowCount) {
+        const dataTable = $container.find('table').dataTable({
             searching: false,
-            pageLength: 50,
+            pageLength: dataTablePageLength,
             lengthChange: false,
             columnDefs: [
                 {
+                    // action row should not be orderable
                     targets: 2,
                     orderable: false,
                 },
+                {
+                    // logs row should not be orderable
+                    targets: 3,
+                    orderable: false,
+                },
             ],
-            fnDrawCallback: function () {
-                // Hide pagination controls if length is less than 51
-                if (numberOfChildJobs < 51) {
+            fnDrawCallback: () => {
+                // Hide pagination controls if length is less than or equal to table length
+                if (rowCount <= dataTablePageLength) {
                     $('.dataTables_paginate').hide();
                 }
             },
         });
+        return dataTable;
     }
 
-    function factory(config) {
-        const { model } = config,
-            widgets = {};
-        let $container;
-
-        function createTableRow(id) {
-            const jobTable = $container.find('tbody')[0],
-                newRow = document.createElement('tr');
-            newRow.setAttribute('data-element-job-id', id);
-            newRow.classList.add('job-info');
-            newRow.classList.add(`${cssBaseClass}__row`);
-            jobTable.appendChild(newRow);
-            return newRow;
+    function factory() {
+        const widgets = {},
+            widgetsById = {},
+            runtime = Runtime.make(),
+            listeners = [];
+
+        let $container, tableBody;
+
+        function startParamsListener(jobId) {
```

Comment:
New -- add in a listener for job-info so that the parameters can be added to the table (replacing "Child job ...")